icons/gitlab.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m22 13.29-3.33-10a.42.42 0 0 0-.14-.18.38.38 0 0 0-.22-.11.39.39 0 0 0-.23.07.42.42 0 0 0-.14.18l-2.26 6.67H8.32L6.1 3.26a.42.42 0 0 0-.1-.18.38.38 0 0 0-.26-.08.39.39 0 0 0-.23.07.42.42 0 0 0-.14.18L2 13.29a.74.74 0 0 0 .27.83L12 21l9.69-6.88a.71.71 0 0 0 .31-.83Z" />
+  <path d="m22 13.3-3.33-10a.42.42 0 0 0-.73-.05l-2.26 6.67H8.32L6.1 3.26a.38.38 0 0 0-.73-.01L2 13.3a.74.74 0 0 0 .27.82L12 21l9.7-6.88a.7.7 0 0 0 .3-.82"/>
```

Comment:
Two things to keep in mind are that: brand icons are deprecated and will likely be removed #94 & that this isn't even the latest Gitlab icon. :)
```suggestion
  <path d="m22 13.3-3.33-10a.42.42 0 0 0-.73-.05l-2.26 6.67H8.32L6.1 3.26a.38.38 0 0 0-.73-.01L2 13.3a.74.74 0 0 0 .27.82L12 21l9.7-6.88a.7.7 0 0 0 .3-.82" />
```